<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalc</class>
 <widget class="QWidget" name="CreditCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок кредита (мес)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="l_sum_credit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма кредита</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="l_term_credit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="l_percent">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Расчитать тип </string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_ann">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Аннуитетные</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_dif">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Дифференцированные</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="result_text">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>341</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
